src/json/program/AfterUserEventProcessedTriggerContext.schema.ts

```diff
@@ -0,0 +1,30 @@
+import { JSONSchema6 } from "json-schema";
+import {
+  contextTimeField,
+  userContext,
+  customFieldsAndEventsDefinitions
+} from "./ProgramTriggerVariableContext.schema";
+
+const afterUserEventProcessedTriggerContextSchema: JSONSchema6 = {
+  $schema: "http://json-schema.org/draft-06/schema#",
+  type: "object",
+  title: "After User Event Processed Trigger Context",
```

Comment:
Make it more doc friendly?

```suggestion
  title: "Event Trigger Variables",
```